nflows/transforms/splines/rational_quadratic.py

```diff
@@ -139,6 +139,10 @@ def rational_quadratic_spline(
         c = -input_delta * (inputs - input_cumheights)
 
         discriminant = b.pow(2) - 4 * a * c
+
+        float_precision_mask = (torch.abs(discriminant)/(b.pow(2) + 1e-8)) < 1e-6
```

Comment:
Could you add a comment explaining where this particular formula comes from, and define constants/parameters for the magic numbers?

Reply:
I have added a comment trying to explain the logic of the operation without being too verbose.
I am afraid the numbers are manly motivated by heuristics to manage numerical stability in our case.